### PR TITLE
Added Cards

### PR DESCRIPTION
**Pull request by syed-shaida-hussain**
[cards live](https://dreamy-shockley-557c0e.netlify.app/card/card)

### CODE REVIEW

**Review by SHIKHASINGH1506 — APPROVED** [review]
Looks fine to me

File: Card/card.css [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+  width: 15rem;
+}
+
+.card-with-badges-img {
+  border-radius: 5px;
+  padding: 0;
+}
+
+.badge {
+  background-color: rgb(109, 204, 109);
```
Comment (amirnathani9) [inline Card/card.css]: use the color variable instead of directly giving the color

File: Card/card.css [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+  margin-left: 1rem;
+  font-size: 1.5rem;
+}
+
+.product-info {
+  margin-left: 1rem;
+  margin-top: 0.5rem;
+  font-size: 1.3rem;
+  font-weight: 10%;
+  color: rgb(172, 161, 161);
```
Comment (amirnathani9) [inline Card/card.css]: same change here give variable name

**Comment by amirnathani9** [comment]
looks good to me. Add Title, discription , screenshot in discription box.
Also work on the page layout of website

**Pull request merged**